contributors/devel/automation.md

```diff
@@ -38,37 +38,33 @@ The status of the submit-queue is [online.](http://submit-queue.k8s.io/)
 The submit-queue lists what it believes are required on the [merge requirements tab](http://submit-queue.k8s.io/#/info) of the info page. That may be more up to date.
 
 A PR is considered "ready for merging" if it matches the following:
-  * The PR must have the label "cla: yes" or "cla: human-approved"
-  * The PR must be mergeable. aka cannot need a rebase
-  * All of the following github statuses must be green
-     * Jenkins GCE Node e2e
-     * Jenkins GCE e2e
-     * Jenkins unit/integration
+  * The PR must have the label `cncf-cla: yes` or `cla: human-approved`
+  * The PR must be mergeable, aka cannot need a rebase
+  * All of the [Required Github CI Tests](https://github.com/kubernetes/test-infra/blob/master/mungegithub/submit-queue/deployment/kubernetes/configmap.yaml#L15) must be green
```

Comment:
This list is available in a more user friendly version at https://submit-queue.k8s.io/#/info on the Merge Requirements tab. That tab is also dynamically updated, so perhaps it's a better source for this list?